packages/schema/prepack.ts

```diff
@@ -0,0 +1,19 @@
+import fs from "fs";
```

Comment:
Mind putting these two `.ts` files in `scripts/` ? I think that's a good convention for "stuff that's just for working on this package".